docs/admin/scale/gke.md

```diff
@@ -0,0 +1,62 @@
+# Scaling Coder on Google Kubernetes Engine (GKE)
+
+This is a reference architecture for Coder on [Google Kubernetes Engine](#). We regurily load test these environments with a standard [kubernetes example](https://github.com/coder/coder/tree/main/examples/templates/kubernetes) template.
+
+> Performance and ideal node sizing depends on many factors, including workspace image and the [workspace sizes](https://github.com/coder/coder/issues/3519) you wish to give developers. Use Coder's [scale testing utility](./index.md#scale-testing-utility) to test your own deployment.
+
+## 50 users
+
+### Cluster configuration
+
+- **Autoscaling profile**: `optimize-utilization`
+
+- **Node pools**
+  - Default
+    - **Operating system**: `Ubuntu with containerd`
+    - **Instance type**: `e2-highcpu-8`
+    - **Min nodes**: `1`
+    - **Max nodes**: `4`
+
+### Coder settings
+
+- **Replica count**: `1`
+- **Provisioner daemons**: `30`
+- **Template**: [kubernetes example](https://github.com/coder/coder/tree/main/examples/templates/kubernetes)
+- **Coder server limits**:
+  - CPU: `2 cores`
+  - RAM: `4 GB`
+- **Coder server requests**:
+  - CPU: `2 cores`
+  - RAM: `4 GB`
+
+## 100 users
+
+For deployments with 100+ users, we recommend running the Coder server in a separate node pool via taints, tolerations, and nodeselectors.
+
+### Cluster configuration
```

Comment:
If we're using Terraform or GDM for spawning these machines, can we please share and link it here? It might be easier for customers to bring up their own clusters.

Reply:
At this point, we're doing it manually (or internal Terraform configs that are not ready for the public), but agree we should eventually provide the Terraform config for many of these environments.

I'd hate for Terraform to be a prerequisite for us testing a specific environment (e.g. OpenShift, DigitalOcean), but agree that it's highly beneficial for reproducibility and so customers could quickly spin up clusters